src/com/hms_networks/americas/sc/time/TimeConstants.java

```diff
@@ -0,0 +1,39 @@
+package com.hms_networks.americas.sc.time;
+
+/**
+ * Class of constants for the HMS MU Americas Solution Center time library for Ewon Flexy devices.
+ *
+ * @since 1.0
+ * @author HMS Networks, MU Americas Solution Center
+ */
+public class TimeConstants {
```

Comment:
Im wondering about the class name here since its a little misleading. 

TimeConstants.DEFAULT_DEVICE_PASSWORD is kind of a weird thing. 

Maybe not the best change but a little better would be 
TimeLibConstants.java 
that way its more clear that its library constants rather than "time constants" which i would expect to be something more like MILLIS_PER_SECOND or HOURS_PER_DAY

Reply:
I agree, the name is a bit confusing and I went back and forth on using it. I'll name it TimeLibConstants to ensure the class contents make sense.

Reply:
changed